### PR TITLE
Find company logo in production

### PR DESCRIPTION
**Pull request by memotoro**
Hello. 

This is an old issue in production mode. The find_asset method from the assets is not available in production, so it is causing the prawn template to raise an exception. The fix now works for both development and production.

Regards

### CODE REVIEW

**Comment by kennyadsl** [comment]
@memotoro hey can you please make separate commit for the `I18n.t` change and the actual fix?

**Comment by memotoro** [comment]
Hello @kennyadsl 

Sorry for the late reply. I separated two commits, one for the assets in productions and another commit (latest) with the I18n.t fixes.

Let me know if you want me to open a pull request for the I18n.t as well or you can manage it during the merge.

Regards

**Comment by memotoro** [comment]
Hello @aldesantis . I wonder if you could have a look to this PR as it is a while since I submit the fix but it wasn't merge for some previous conflicts. The github robot marked it as `wontfix` for inactivity but the fix is ready to go. Could you check it in the next release? Let me know if anything. Regards

File: app/views/spree/admin/orders/_print.pdf.prawn [hunk 1]
```diff
@@ -4,7 +4,7 @@ require 'prawn/layout'
 
 font @font_face
 
-im = Rails.application.assets.find_asset(Spree::PrintInvoice::Config[:print_invoice_logo_path])
+im = Sprockets::Railtie.build_environment(Rails.application).find_asset(Spree::PrintInvoice::Config[:print_invoice_logo_path])
```
Comment (aldesantis) [inline app/views/spree/admin/orders/_print.pdf.prawn]: Is there a reason you can't use `asset_path` for this? Just asking, I'm not familiar with this API.

Comment (memotoro) [reply app/views/spree/admin/orders/_print.pdf.prawn]: Hi @aldesantis . `Rails.application.assets` is nil in production mode as the assets are served by `sprockets` and are compiled already. For the `prawn` templates we need the actual file object to get the path locally and we might need some context of the `Rails.root` and how the app folders are in the project. In this way we could get the asset file.

Comment (aldesantis) [reply app/views/spree/admin/orders/_print.pdf.prawn]: @memotoro ah, sorry, I didn't realize how it was being used in that template.

**Comment by aldesantis** [comment]
@memotoro one last thing: could you squash your commits into one please? 🙏

**Pull request closed without merging**